ogr/ogrsf_frmts/mvt/mvtutils.h

```diff
@@ -60,23 +60,30 @@
     "description='Number of units in a tile'/>"                                \
     "  <Option name='BUFFER' scope='vector' type='unsigned int' default='80' " \
     "description='Number of units for geometry buffering'/>"                   \
-    "  <Option name='COMPRESS' scope='vector' type='boolean' description="     \
-    "'Whether to deflate-compress tiles' default='YES'/>"                      \
-    "  <Option name='TEMPORARY_DB' scope='vector' type='string' description='" \
-    "Filename with path for the temporary database'/>"                         \
     "  <Option name='MAX_SIZE' scope='vector' type='unsigned int' min='100' "  \
     "default='500000' "                                                        \
     "description='Maximum size of a tile in bytes'/>"                          \
     "  <Option name='MAX_FEATURES' scope='vector' type='unsigned int' "        \
     "min='1' default='200000' "                                                \
     "description='Maximum number of features per tile'/>"
 
+#define MVT_MBTILES_COMMON_DSCO                                                \
+    MVT_MBTILES_PMTILES_COMMON_DSCO                                            \
+    "  <Option name='COMPRESS' scope='vector' type='boolean' description="     \
+    "'Whether to deflate-compress tiles' default='YES'/>"                      \
```

Comment:
In [other places](https://gdal.org/drivers/vector/mvt.html) the GDAL docs refer to DEFLATE/gzip interchangeably though the [PMTiles spec](https://github.com/protomaps/PMTiles/blob/main/spec/v3/spec.md) only defines gzip as a well-known option. The requirements are driven by the [Web DecompressionStream standard](https://developer.mozilla.org/en-US/docs/Web/API/DecompressionStream/DecompressionStream) which on most platforms does support raw `deflate` but I haven't found a compelling reason to support both.